Resources/Prototypes/tags.yml

```diff
@@ -1014,6 +1014,9 @@
 - type: Tag
   id: TrashBag
 
+- type: Tag
+  id: CannonRestrict
+
 - type: Tag
```

Comment:
Alphabetical you savage.

Reply:
Wtf I put it alphabetical.... I blame the ghost